src/workloads/query/TimeseriesStressUnpacking.yml

```diff
@@ -0,0 +1,276 @@
+      TemplateParameters:
+        Name: "match_10_buckets_include_10_fields"
+        Active: [5]
+        # With 988 documents per bucket, 10 buckets will have 'measurement' <= 9880 (988 * 10).
+        Predicate: { $lte: 9880 }
+        FieldsToProject: *Project10Fields
+
+  - ActorFromTemplate:
+      TemplateName: RunMatchPipeline
+      TemplateParameters:
+        Name: "match_10_buckets_exclude"
+        Active: [6]
+        # With 988 documents per bucket, 10 buckets will have 'measurement' <= 9880 (988 * 10).
+        Predicate: { $lte: 9880 }
+
+  - ActorFromTemplate:
+      TemplateName: RunMatchProjectPipeline
+      TemplateParameters:
+        Name: "match_100_buckets_include_1_field"
+        Active: [7]
+        # With 988 documents per bucket, 100 buckets will have 'measurement' <= 98800 (988 * 100).
+        Predicate: { $lte: 98800 }
+        FieldsToProject: *Project1Field
+
+  - ActorFromTemplate:
+      TemplateName: RunMatchProjectPipeline
+      TemplateParameters:
+        Name: "match_100_buckets_include_10_fields"
+        Active: [8]
+        # With 988 documents per bucket, 100 buckets will have 'measurement' <= 98800 (988 * 100).
+        Predicate: { $lte: 98800 }
+        FieldsToProject: *Project10Fields
+
+  - ActorFromTemplate:
+      TemplateName: RunMatchPipeline
+      TemplateParameters:
+        Name: "match_100_buckets_exclude"
+        Active: [9]
+        # With 988 documents per bucket, 100 buckets will have 'measurement' <= 98800 (988 * 100).
+        Predicate: { $lte: 98800 }
+        # Without a $project stage, $_internalUnpackBucket will have an "exclude" parameter.
+
+  - ActorFromTemplate:
+      TemplateName: RunMatchProjectPipeline
+      TemplateParameters:
+        Name: "match_1000_buckets_include_1_field"
+        Active: [10]
+        # With 988 documents per bucket, 1000 buckets will have 'measurement' <= 988000 (988 * 1000).
+        Predicate: { $lte: 988000 }
+        FieldsToProject: *Project1Field
+
+  - ActorFromTemplate:
+      TemplateName: RunMatchProjectPipeline
+      TemplateParameters:
+        Name: "match_1000_buckets_include_10_fields"
+        Active: [11]
+        # With 988 documents per bucket, 1000 buckets will have 'measurement' <= 988000 (988 * 1000).
+        Predicate: { $lte: 988000 }
+        FieldsToProject: *Project10Fields
+
+  - ActorFromTemplate:
+      TemplateName: RunMatchPipeline
+      TemplateParameters:
+        Name: "match_1000_buckets_exclude"
+        Active: [12]
+        # With 988 documents per bucket, 1000 buckets will have 'measurement' <= 988000 (988 * 1000).
+        Predicate: { $lte: 988000 }
+        # Without a $project stage, $_internalUnpackBucket will have an "exclude" parameter.
+
+AutoRun:
+  - When:
+      mongodb_setup:
+        $eq:
+          - standalone-all-feature-flags
+          - standalone
+      branch_name:
+        $neq:
+          - v4.0
+          - v4.2
+          - v4.4
+          - v5.0
```

Comment:
Can we also disallow `v6.0`? I think keeping `v7.0` makes sense so that we can do master vs 7.0 comparison, but I don't see much benefit in running these workloads on 6.0.

Reply:
there was quite a bit of work for time-series in 7.0, no? given the scarcity of coverage why no run these workloads in 6.0 to see the cross-version trend?